_data/analysis-files.yml

```diff
@@ -0,0 +1,36 @@
+- file: continent-estimated.csv
+  name: Estimated Failure Rates By Continent
+  desc: The estimated failure rates of data portals and journal websites grouped by continents
```

Comment:
Given that these lean more towards sentences in terms of length, should we make these proper sentences and include periods?

Reply:
Great suggestion! It now starts with "This data represents/includes ..."